packages/calcite-components/src/components/rating/rating.tsx

```diff
@@ -259,31 +239,26 @@ export class Rating
           <fieldset class="fieldset" disabled={this.disabled}>
             <legend class="visually-hidden">{this.messages.rating}</legend>
             {this.starsMap.map(
-              ({
-                average,
-                checked,
-                focused,
-                fraction,
-                hovered,
-                id,
-                idx,
-                partial,
-                selected,
-                value,
-              }) => {
+              ({ average, checked, fraction, hovered, id, partial, selected, value, tabIndex }) => {
                 return (
                   <label
                     class={{
                       star: true,
-                      focused,
                       selected,
                       hovered,
                       average,
                       partial,
                     }}
+                    data-value={value}
                     htmlFor={id}
+                    onClick={this.handleLabelClick}
```

Comment:
could this be done in `pointerDown`?

Reply:
preventing default in pointerDown handler wouldn't stop `click` event from being invoked.

Reply:
Interesting. I thought this was a browser bug, but it seems to be working as intended (see _Pointer Event: Suppress compatibility mouse events on click_ [test](https://github.com/web-platform-tests/wpt/blob/master/pointerevents/pointerevent_suppress_compat_events_on_click.html) and [results](https://w3c.github.io/test-results/pointerevents/all.html#test-file-48). Can you add a comment about this for posterity?